### ISSUE TITLE
Release the gazelle plugin in the BCR

### ISSUE THREAD

**Issue by shs96c** [bazelbuild/bazel-skylib#420]
It would be useful if there were a version of the gazelle plugin in the BCR, as this would unblock progress on making [contrib_rules_jvm](https://github.com/bazel-contrib/rules_jvm) available for `bzlmod` users.

**Comment by tetromino** [comment]
Duplicate of #410
